### PR TITLE
First time signup flow

### PR DESCRIPTION
**Pull request by phawk**
- When a user signs up on openhq.io they just enter team name and their email address
- User receives confirmation email with a link to setup their team/account
- Step 1 user completes their user info
- Step 2 user creates first project
- Step 3 user invites their team

### CODE REVIEW

**Comment by phawk** [comment]
@colinyoung87 forgot to say about this PR, take a look at it when you get a chance over the next couple days

File: app/controllers/setup_controller.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      @project.users << current_user
+
+      redirect_to setup_invite_team_path
+    else
+      render :first_project
+    end
+  end
+
+  def invite_team
+    @team_invites = TeamInvitesForm.new(current_team, current_user)
```
Comment (colinyoung87) [inline app/controllers/setup_controller.rb]: does this not need to know the `project_ids`?

File: app/views/signups/success.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<h1>You’re team has been created, check your email</h1>
```
Comment (colinyoung87) [inline app/views/signups/success.html.erb]: Your*

**Comment by colinyoung87** [comment]
:+1:

**Pull request merged**